docs/source/models.rst

```diff
@@ -481,8 +483,11 @@ a model with random weights by calling its constructor:
     resnet18 = models.quantization.resnet18()
     resnet50 = models.quantization.resnet50()
     resnext101_32x8d = models.quantization.resnext101_32x8d()
+    resnext101_64x4d = models.quantization.resnext101_64x4d()
```

Comment:
We should have a similar line on the non-quantized section of the doc. Github won't let me add a comment directly but it should be around line 63.

Reply:
Ah, seems like I missed it. I usually search the location of text `resnext101` and add the new model after, but seems like we miss the `resnext101_32x8d` on the non-quantized part. The new commit would add both.